packages/react-devtools/app.js

```diff
@@ -15,7 +15,10 @@ var updateNotifier = require('update-notifier');
 var pkg = require('./package.json');
 
 var mainWindow = null;
-var argv = process.argv.slice(2);
```

Comment:
Hmm, I'm pretty sure it worked. Isn't calling `slice(2)` twice fine because `bin.js` spawns a new process (which has its own `argv`)?